src/quo/components/settings/data_item/view.cljs

```diff
@@ -44,18 +44,18 @@
     subtitle]])
 
 (defn- left-title
-  [{:keys [title label size theme]}]
+  [{:keys [title label size blur? theme]}]
```

Comment:
We can let `(style/title)` to extract `blur?` and `theme` on its own. This way we will avoid polluting view with unnecessary vars and also exclude the problem when someone pass arguments to `(style/title)` in incorrect order 
Likes this:

```clojure
(defn- left-title
  [{:keys [title label size ] :as props}]
  ...
  text/text
    {:weight :regular
     :size   :paragraph-2
     :style  (style/title props)}  <---
    title]
    ...)
```